containers/Dockerfile

```diff
@@ -1,7 +1,5 @@
 # Dockerfile for kraken
 
-FROM quay.io/openshift/origin-tests:latest as origintests
```

Comment:
I dont think you can delete this

Reply:
We used to use this image to copy the kubectl and oc clients, we are now installing them from a different source. Thoughts?

Reply:
Ahh I see now, didnt see them at the bottom